sdk/.env.example

```
@@ -1,8 +1,22 @@
+# Mnemonics used in the examples (don't ever use those for real funds)
 NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_1="endorse answer radar about source reunion marriage tag sausage weekend frost daring base attack because joke dream slender leisure group reason prepare broken river"
 NON_SECURE_USE_OF_DEVELOPMENT_MNEMONIC_2="width scatter jaguar sponsor erosion enable cave since ancient first garden royal luggage exchange ritual exotic play wall clinic ride autumn divert spin exchange"
+# Seeds used in the examples (don't ever use those for real funds)
 NON_SECURE_USE_OF_DEVELOPMENT_SEED_1=0x256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b2
 NON_SECURE_USE_OF_DEVELOPMENT_SEED_2=0x256a818b2aac458941f7274985a410e57fb750f3a3a67969ece5bd9ae7eef5b3
+# The account aliases used in the examples
+ACCOUNT_ALIAS_1="Alice"
+ACCOUNT_ALIAS_2="Bob"
+# The wallet database folder used in the examples
+WALLET_DB_PATH="./example.walletdb"
```

maybe
```suggestion
WALLET_DB_PATH="./example-walletdb"
```
because it's a folder and not a file

---

yeah, sure